### PR TITLE
D2IQ-61004: Adding etcd metrics on telegraf

### PR DESCRIPTION
**Pull request by wavesoft**
## High-level description

Adds the etcd metrics endpoint as one of the scraping sources on telegraf.

## Corresponding DC/OS tickets (required)

  - [D2IQ-61004](https://jira.d2iq.com/browse/D2IQ-61004) Include etcd in the dc/os monitoring, add dashboards.

## Related tickets (optional)

_(none)_

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -2605,6 +2605,20 @@ package:
         # insecure_skip_verify = true
         [inputs.zookeeper.tags]
           dcos-component-name = "ZooKeeper"
+      # Reads metrics from ETCs's prometheus client
+      [[inputs.prometheus]]
+        ## An array of urls to scrape metrics from.
+        urls = ["http://localhost:2379/metrics"]
```
Comment (janisz) [inline gen/dcos-config.yaml]: Can you add a test for it like this https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/test_metrics.py#L476

Comment (wavesoft) [reply gen/dcos-config.yaml]: Yup, that's what I am writing now 😄

**Comment by wavesoft** [comment]
@mesosphere-mergebot label Ready For Review

File: gen/dcos-config.yaml [hunk 2]
```diff
@@ -2609,6 +2609,20 @@ package:
         # insecure_skip_verify = true
         [inputs.zookeeper.tags]
           dcos-component-name = "ZooKeeper"
+      # Reads metrics from ETCs's prometheus client
```
Comment (orlandohohmeier) [inline gen/dcos-config.yaml]: 🔡  etcd's

File: packages/dcos-integration-test/extra/test_metrics.py [hunk 3]
```diff
@@ -185,6 +185,21 @@ def check_cockroachdb_metrics():
     check_cockroachdb_metrics()
 
 
+@pytest.mark.supportedwindows
+def test_metrics_master_etcd(dcos_api_session):
+    """Assert that DC/OS ETCD metrics on master are present."""
```
Comment (orlandohohmeier) [inline packages/dcos-integration-test/extra/test_metrics.py]: 🔠 etcd

**Comment by wavesoft** [comment]
@mesosphere-mergebot sync

**Comment by orlandohohmeier** [comment]
Adding the ship it label without waiting for CI as test were green and the last changes only corrected typos in comments and added a change-log entry. @mesosphere-mergebot label [Ship It]

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ship-it

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ship-it

**Pull request merged**